src/services/lnd/index.ts

```diff
@@ -87,6 +87,27 @@ export const LndService = (
     }
   }
 
+  const getChannelBalances = async (pubkey?: Pubkey) => {
```

Comment:
I think the function name is not accurate. we're getting aggregate balance here, not balance per channel. 

Also getBalance typically return a Satoshis amount, here we are returning  really inbound/output. so maybe `getInboundOutbalanceBalance`?

Reply:
renamed to getInboundOutboundBalance